CurrencyExchangeService/pom.xml

```diff
@@ -106,5 +106,6 @@
 			</plugin>
 		</plugins>
 	</build>
-
+ 
```

Comment:
Is this extra space necessary, please remove extra line and follow standard practices

Reply:
will change this